### PR TITLE
chore(deps): Update graphql dependencies

### PR DESCRIPTION
**Pull request by bastilian**

### CODE REVIEW

**Comment by akofink** [comment]
I'm seeing the following error on the system details page:

```
Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `ComplianceRemediationButton`.
    in ComplianceRemediationButton (created by ConnectFunction)
    in ConnectFunction (created by SystemRulesTable)
    in div (created by ToolbarItem)
    in ToolbarItem (created by SystemRulesTable)
    in div (created by Context.Consumer)
    in div (created by ToolbarContent)
    in ToolbarContent (created by PrimaryToolbar)
    in div (created by Toolbar)
    in Toolbar (created by PrimaryToolbar)
    in PrimaryToolbar (created by SystemRulesTable)
    in SystemRulesTable (created by SystemQuery)
    in SystemQuery (created by SystemDetails)
    in SystemDetails (created by WrappedSystemDetails)
    in WrappedSystemDetails (created by SystemDetails)
    in section (created by Context.Consumer)
    in Main (created by ConnectFunction)
    in ConnectFunction (created by SystemDetails)
    in StateViewPart (created by SystemDetails)
    in StateView (created by StateViewWithError)
    in StateViewWithError (created by SystemDetails)
    in SystemDetails (created by Router)
    in Route (created by Router)
    in Switch (created by Router)
    in Suspense (created by Router)
    in Router (created by Routes)
    in Routes (created by App)
    in App (created by RouterParams)
    in RouterParams (created by ConnectFunction)
    in ConnectFunction (created by Context.Consumer)
    in withRouter(Connect(RouterParams))
    in ApolloProvider
    in Router (created by BrowserRouter)
    in BrowserRouter
    in IntlProvider
    in Provider react_devtools_backend.js:2430:23

Uncaught Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `ComplianceRemediationButton`.
    React 16
    unstable_runWithPriority scheduler.development.js:653
    React 5
    promise callback*onNewData react-hooks.esm.js:366
    next react-hooks.esm.js:225
    notifySubscription Observable.js:135
    onNotify Observable.js:179
    next Observable.js:235
    iterateObserversSafely bundle.esm.js:435
    iterateObserversSafely bundle.esm.js:435
    next bundle.esm.js:410
    invoke bundle.esm.js:1209
    queryListenerForObserver bundle.esm.js:1294
    broadcastQueries bundle.esm.js:1559
    broadcastQueries bundle.esm.js:1557
    broadcastQueries bundle.esm.js:1555
    subscription bundle.esm.js:1646
    next Observable.js:322
    notifySubscription Observable.js:135
    onNotify Observable.js:179
    next Observable.js:235
    next bundle.esm.js:866
    next bundle.esm.js:866
    notifySubscription Observable.js:135
    onNotify Observable.js:179
    next Observable.js:235
    createHttpLink bundle.esm.js:76
    promise callback*createHttpLink/</< bundle.esm.js:75
    Subscription Observable.js:197
    subscribe Observable.js:279
    multiplex bundle.esm.js:864
    Subscription Observable.js:197
    subscribe Observable.js:279
    getObservableFromLink bundle.esm.js:1595
    fetchRequest bundle.esm.js:1630
    fetchRequest bundle.esm.js:1629
    fetchQuery bundle.esm.js:1146
    step tslib.es6.js:100
    verb tslib.es6.js:81
    __awaiter tslib.es6.js:74
    __awaiter tslib.es6.js:70
    fetchQuery bundle.esm.js:1092
    observeQuery bundle.esm.js:1460
    setUpQuery bundle.esm.js:388
    onSubscribe bundle.esm.js:366
    _this bundle.esm.js:96
    Subscription Observable.js:197
    subscribe Observable.js:279
    node_modules vendor.js:380
    getExecuteResult react-hooks.esm.js:74
    node_modules vendor.js:271
    result react-hooks.esm.js:383
    useDeepMemo react-hooks.esm.js:349
    useBaseQuery react-hooks.esm.js:383
    useQuery react-hooks.esm.js:403
    SystemDetails index.js:1
    renderWithHooks React
react-dom.development.js:23965
```

File: packages/inventory-compliance/package.json [hunk 1]
```diff
@@ -44,9 +44,8 @@
         "@redhat-cloud-services/frontend-components-remediations": "*",
         "@redhat-cloud-services/frontend-components-utilities": "*",
         "@apollo/react-hooks": "^3.1.5",
-        "apollo-boost": "^0.1.23",
-        "graphql-tag": "^2.10.0",
-        "react-apollo": "^2.3.3",
```
Comment (vkrizan) [inline packages/inventory-compliance/package.json:49]: What's behind the `react-apollo` removal?

**Review by akofink — APPROVED** [review]
Ok, sorry - it seems to work as expected. The RemediationsButton issue isn't expected to be fixed with this. Thanks for the clarification on slack, @bastilian! :heavy_check_mark:

File: packages/inventory-compliance/package.json (hunk 1 above)
Comment (bastilian) [reply packages/inventory-compliance/package.json:49]: We don't need it anymore `react-apollo` used to be the overarching Apollo package, but we have been moving to `@apollo/*` packages over time.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**